docs/csharp/programming-guide/main-and-command-args/top-level-statements.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Top-level statements - C# Programming Guide"
+description: Learn about top-level statements in C# 9 and later.
+ms.date: 03/08/2021
+helpviewer_keywords:
+  - "C# language, top-level statements"
+  - "C# language, Main method"
+---
+# Top-level statements (C# Programming Guide)
+
+Starting in C# 9, you don't have to explicitly include a `Main` method in a console application project. Instead, you can use the *top-level statements* feature to minimize the code you have to write. In this case, the compiler implicitly provides a class and `Main` method entry point for the application.
+
+Here's a *Program.cs* file that is a complete C# program in C# 9:
+
+:::code language="csharp" source="snippets/top-level-statements-1/Program.cs":::
+
+Top-level statements let you write simple programs for small utilities such as Azure Functions and GitHub Actions. They also make it simpler for new C# programmers to get started learning and writing code.
```

Comment:
This use of "simple programs" is fine because it does flow naturally with the concept. I like how you've done this Tom.